geles-frontend/src/EditOrder.svelte

```diff
@@ -0,0 +1,517 @@
+    let errString: string = "";
+    if (reason.response) {
+      errString =
+        reason.response.data +
+        " " +
+        reason.response.status +
+        " " +
+        reason.response.headers;
+    } else if (reason.request) {
+      let errString = reason.request;
+    } else {
+      // Something happened in setting up the request that triggered an Error
+      let errString = reason;
+    }
+    console.log("Error", reason);
+    addNotification(errString, AppNotificationType.DANGER);
+  }
+
+  async function getOrderData() {
+    let resp = await axios.get(`/orders/${id}`, { withCredentials: true });
+    order = resp.data;
+    editDto.orderFlowers = order.orderFlowers;
+    editDto.address = order.address;
+    editDto.contactPhone = order.contactPhone;
+    editDto.version = order.version;
+    recalcSum();
+  }
+
+  function handleUpdate() {
+    axios.put(`/orders/${id}/edit`, editDto)
+    .then(getOrderData)
+    .then(() => addNotification("Užsakymas atnaujintas",AppNotificationType.SUCCESS))
+    .catch(handleError);
+  }
+
+  function handlePay() {
+    axios
+      .post(`/orders/${id}/pay`, { version: order.version })
+      .then(getOrderData)
+      .then(() => addNotification("Užsakymas apmokėtas",AppNotificationType.SUCCESS))
+      .catch(handleError);
+  }
+
+  function handleCancel() {
+    axios
+      .post(`/orders/${id}/cancel`, { version: order.version })
+      .then(getOrderData)
+      .then(() => addNotification("Užsakymas atšauktas",AppNotificationType.SUCCESS))
+      .catch(handleError);
+  }
+  function handleConfirmOrder() {
+    axios
+      .post(`/orders/${id}/confirm`, { version: order.version })
+      .then(getOrderData)
+      .then(() => addNotification("Užsakymas patvirtintas",AppNotificationType.SUCCESS))
+      .catch(handleError);
+  }
+  function handleConfirmDelivered() {
+    axios
+      .post(`/orders/${id}/confirmDelivery`, { version: order.version })
+      .then(getOrderData)
+      .then(() => addNotification("Užsakymo pristatymas patvirtintas",AppNotificationType.SUCCESS))
+      .catch(handleError);
+  }
+
+  function formatDate(date: any) {
+    var d = new Date(date),
+      month = "" + (d.getMonth() + 1),
+      day = "" + d.getDate(),
+      year = d.getFullYear();
+
+    if (month.length < 2) month = "0" + month;
+    if (day.length < 2) day = "0" + day;
+
+    return [year, month, day].join("-");
+  }
+  function orderStatusString(status: any): string {
+    switch (status) {
+      case "UNPAID":
+        return "Neapmokėta";
+      case "PAID":
+        return "Apmokėtas";
+      case "CONFIRMED":
+        return "Patvirtintas";
+      case "DELIVERED":
+        return "Pristatytas";
+      case "CANCELED":
+        return "Atšauktas";
+      default:
+        return "???????";
+    }
+  }
+
+  // Run code on component mount (once)
+  onMount(async () => {
+    await axios.get(`/flowers/`).then(resp => {
+      flowers = resp.data;
+    });
```

Comment:
This `await` is unnecessary, remove `await` and use assignment operator and two statements.